wee_slack.py

```diff
@@ -2957,6 +3100,8 @@ def linkify_text(message, team, channel):
             named = targets.groups()
             if named[1] in ["group", "channel", "here"]:
                 message[item[0]] = "<!{}>".format(named[1])
+            elif named[1] in list(channel.team.subteam_handles_to_id.viewkeys()):
+                message[item[0]] = "<!subteam^{}|{}>".format(channel.team.subteam_handles_to_id[named[1]], named[1])
```

Comment:
If you write e.g. colon after the group, it will be lost because `named[2]` is not included. I see this was an issue with group/channel/here too, I've fixed that in master now.